.github/workflows/publish-docker-images.yaml

```diff
@@ -0,0 +1,81 @@
+name: Publish Docker Images
+
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    # The branches below must be a subset of the branches above.
+    branches: [main, master]
+  release:
```

Comment:
I thought that the Action runner woul be enabled in case of the 'release' event only.
Do we have to add the 'push' and 'pull_request'  event?